source/api/commands/state.md

```diff
@@ -0,0 +1,148 @@
+---
+title: state
+
+---
+
+Get and set state _synchronously_.
+
+# Syntax
+
+```javascript
+.state()
+.state(key)
+.state(key, value)
+```
+
+## Usage
+
+### Getting
+
+```javascript
+var window = cy.state("window")
```

Comment:
Any code examples should use single quotes instead of double quotes. We think this makes it more copy/paste-able for most users.

Reply:
Aw, cheers. Can’t believe I didn’t spot that 😳 I’ll sort it shortly.

Reply:
All quotes corrected.

> I also reformatted some of the headers to use `###` instead of `***` in light of a recent commit to the `develop` branch that I rebased from.

<img width="357" alt="screen shot 2018-07-30 at 19 21 39" src="https://user-images.githubusercontent.com/1855109/43415607-f01c3a80-942d-11e8-8148-18ff2ac21c7a.png">